test/IntegrationTests/MSTest.Acceptance.IntegrationTests/AnalyzersTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Testing.Platform.Acceptance.IntegrationTests;
+using Microsoft.Testing.Platform.Acceptance.IntegrationTests.Helpers;
+
+namespace MSTest.Acceptance.IntegrationTests;
+
+[TestClass]
+public sealed class AnalyzersTests : AcceptanceTestBase<NopAssetFixture>
+{
+    [TestMethod]
+    public async Task AnalyzerMessagesShouldBeLocalized()
+    {
+        string code = """
+#file Analyzers.csproj
+<Project Sdk="MSTest.Sdk/$MSTestVersion$">
+
+  <PropertyGroup>
+    <TargetFrameworks>$TargetFrameworks$</TargetFrameworks>
+    <RunAnalyzers>true</RunAnalyzers>
+  </PropertyGroup>
+
+</Project>
+
+#file UnitTest1.cs
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[TestClass]
+public class UnitTest1
+{
+    [DataRow(0)]
+    public void TestMethod()
+    {
+    }
+}
+""".PatchTargetFrameworks(TargetFrameworks.NetCurrent)
+    .PatchCodeWithReplace("$MSTestVersion$", MSTestVersion);
+
+        using TestAsset testAsset = await TestAsset.GenerateAssetAsync("Analyzers", code);
+        DotnetMuxerResult result = await DotnetCli.RunAsync($"build {testAsset.TargetAssetPath}", AcceptanceFixture.NuGetGlobalPackagesFolder.Path, environmentVariables: new()
+        {
+            ["DOTNET_CLI_UI_LANGUAGE"] = "it-IT",
```

Comment:
Why are you setting the 3 vars? Only one should be working, right?

Reply:
@Evangelink We [propagate the *existing* env variables to DotnetCli](https://github.com/microsoft/testfx/blob/2564c52733fde212f367e5869abdc12046c6c8e5/test/Utilities/Microsoft.Testing.TestInfrastructure/DotnetCli.cs#L64-L65). I didn't spend the time to check which one takes precedence as I thought it's okay to just set everything.